### PR TITLE
Alphabetize all languages besides English

### PR DESCRIPTION
**Pull request by Shobha-go**
### Description
Reordered the languages in the configuration file to alphabetize all languages

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)
Fixes [#1253](https://github.com/seattle-uat/civiform/issues/1253) Alphabetize all languages besides English

@neel-a-mehta

### CODE REVIEW

File: universal-application-tool-0.0.1/conf/application.conf [hunk 1]
```diff
@@ -105,7 +105,7 @@ play.modules {
 # to map through to internal messages, or allow the language to be stored in a cookie.
 play.i18n {
   # The application languages.
-  langs = [ "en-US", "zh-TW", "es-US", "vi", "so", "am", "ko", "tl" ]
```
Comment (carolinedanzi) [inline universal-application-tool-0.0.1/conf/application.conf:108]: suggestion (blocking): Can we add a comment here to explain these should be alphabetized according to their English names?

**Review by carolinedanzi — COMMENTED** [review]
Thanks for doing this! One nit comment, and then I think it's good

File: universal-application-tool-0.0.1/conf/application.conf (hunk 1 above)
Comment (Shobha-go) [reply universal-application-tool-0.0.1/conf/application.conf:108]: Thank you for the suggestion @carolinedanzi, I have added the comments and committed the changes here, please review and approve it.

Comment (carolinedanzi) [reply universal-application-tool-0.0.1/conf/application.conf:108]: Thank you for making the change!

**Review by carolinedanzi — APPROVED** [review]

**Pull request merged**